apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -152,13 +152,13 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
         {isLoading && !slots.length && (
           <>
             <SkeletonContainer className="mb-2">
-              <SkeletonText width="full" height="20" />
+              <SkeletonText className="h-5 w-full" />
```

Comment:
The new skeleton components don't support the width and height props anymore, rather they expect you to use className for it. Added that everywhere.